### PR TITLE
change header semantics

### LINKED ISSUE #171: Adjust title header tags
All of header tags should be semantically ordered.

Check [here](https://wave.webaim.org/report#/https://die-lernwerkstatt.org/) for more details.

### PR DESCRIPTION
**Pull request by hikes101**
**What**:
- change header styles
- semantically order header tags

**Why**:
- follow best practice

**Checklist**:

- [ ] Documentation added to the
      [README](https://github.com/Lernwerkstatt/site/blob/master/README.md)
- [x] Tests
- [ ] Ready to be merged
      

Fixes #171

### CODE REVIEW

**Comment by hikes101** [comment]
@pheeria @jarjan could you please have a look at this? not sure if it's what you wanted

File: src/views/partials/home/modal.hbs [hunk 1]
```diff
@@ -12,9 +12,9 @@
         </div>
       </div>
       <div class="modal-body text-dark">
-        <h4 class="mb-3">
+        <p class="h3 mb-3">
           {{ name }}
-        </h4>
+        </>
```
Comment (jarjan) [inline src/views/partials/home/modal.hbs]:
```suggestion
        </p>
```

File: static/css/style.css [hunk 2]
```diff
@@ -227,3 +227,15 @@ main {
   -moz-column-gap: 40px;
   column-gap: 40px;
 }
+
+h1, .h1 {
+  font-size: 2.14375rem; }
```
Comment (jarjan) [inline static/css/style.css]: Doesn't bootstrap already have this styles for h1, h2, ...?

Comment (hikes101) [reply static/css/style.css]: They are slightly different. I downsized them a bit because we used a lot of h3, h4, h5. So the old h3 is now h2...

**Pull request merged**